usaspending_api/etl/tests/test_executive_compensation_etl.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import json
+
+from django.core.management import call_command
+from usaspending_api.references.models import LegalEntityOfficers
+
+from model_mommy import mommy
+
+
+@pytest.fixture
+def test_exec_etl_fixture():
```

Comment:
I always get confused about the role of etl_test_data.json, and it's especially weird in a case like that depends partially on data generated here, but partially on data from the "virtual broker database" represented by etl_test_data.json.  I feel like a comment to that effect would be helpful (even if that comment appeared in multiple test files.)